cypress/e2e/basic-starter.cy.ts

```diff
@@ -19,4 +19,102 @@ describe('Basic Starter', () => {
         });
     });
   });
+
+  it('has working footer menu links', () => {
+    const links = ['Home', 'Articles', 'Events', 'People', 'Places'];
+    links.forEach((link) => {
+      cy.get('.footer-menu')
+        .contains(link)
+        .invoke('attr', 'href')
+        .then((href) => {
+          cy.visit(href);
+          cy.title().should('eq', `${link} - Acquia CMS`);
```

Comment:
Ugh, sorry, my brain was not functioning. I was suggesting using `be` instead of `eq`. That said: this is _totally_ a nitpick so feel free to disregard!